src/components/MultiHopTrade/components/TradeConfirm/TradeConfirmFooter.tsx

```diff
@@ -251,6 +261,23 @@ export const TradeConfirmFooter: FC<TradeConfirmFooterProps> = ({
             </Skeleton>
           </Row.Value>
         </Row>
+        <Row>
+          <Row.Label>
+              <Text translation='trade.recipientAddress' />
+            </Row.Label>
+            <Row.Value>
+              <HStack>
+                <RawText>{middleEllipsis(receiveAddress ?? '')}</RawText>
+                <Link
+                  href={`${sellAsset.explorerAddressLink}${receiveAddress}`}
+                  isExternal
+                  aria-label='View on block explorer'
+                >
+                  <Icon as={ExternalLinkIcon} />
+                </Link>
+              </HStack>
+            </Row.Value>
+          </Row>
```

Comment:
Let's create a reusable component:

Either `TradeRecipientAddressRow` which is selecting the recipient address with redux (no props) and consume it in both `TradeConfirmFooter` and `TradeConfirmSummary`

or 

A dumb component, without the selectors, that is taking sellAsset, recipientAddress as props, and consume it in both components too

I would be happy with both choice, the second one might be better because of reusability outside of swapper umbrella